dockerizeit/master/globalconfig.groovy

```diff
@@ -20,9 +20,14 @@ Jenkins.instance.setScmCheckoutRetryCount(properties.global.scmCheckoutRetryCoun
 
 // Change it to the DNS name if you have it
 println "--> setting jenkins root url"
-def ip = InetAddress.localHost.getHostAddress()
 jlc = JenkinsLocationConfiguration.get()
-jlc.setUrl("http://$ip:8080")
+if (properties.global.jenkinsRootUrl) {
+  println "--> ${properties.global.jenkinsRootUrl}"
+  jlc.setUrl(properties.global.jenkinsRootUrl)
+} else {
+  def ip = InetAddress.localHost.getHostAddress()
+  jlc.setUrl("http://$ip:8080")
```

Comment:
Please add a print out here. I understand that it wasn't there in the first place but let's make code better if we touch it